patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -61,7 +61,7 @@ index 64bea5349783a7a0b13540bd40662b4575437ca3..9c88e501f0a1b953fce8183fa2b4ad3e
                  <action android:name="android.web.action.APP_ATTRIBUTION" />
                  <category android:name="android.intent.category.DEFAULT" />
              </intent-filter>
-+            <intent-filter><action android:name="android.intent.action.SEND" /><category android:name="android.intent.category.DEFAULT" /><data android:mimeType="text/plain" /><data android:mimeType="multipart/related" /><data android:mimeType="message/rfc822" /></intent-filter>
++            <intent-filter><action android:name="android.intent.action.WEB_SEARCH" /><category android:name="android.intent.category.DEFAULT" /></intent-filter><intent-filter><action android:name="android.intent.action.SEND" /><category android:name="android.intent.category.DEFAULT" /><data android:mimeType="text/plain" /><data android:mimeType="multipart/related" /><data android:mimeType="message/rfc822" /></intent-filter>
```

Comment:
This is getting long, maybe it is better changed into a `jinja2` template?

Reply:
We already have an issue to reorganize `AndroidManifest.xml` https://github.com/brave/brave-browser/issues/15942
But since it's not a one-liner I would leave this change as is for this particular issue.
If you see a quick via `jinja2`, please elaborate as it's not obvious for me.

Reply:
Ok I didn't realize there is an issue for that already. But in this specific case, I suppose the whole xml node can be extracted to a separate file and be included, similar to `{% include "//brave/android/java/AndroidManifest.xml" %}`?

Reply:
Since there is a separate issue I agree that this can be addressed later in that issue instead of here.

Reply:
Agreed, for now I did move our intent filters to a separate xml file.